Cargo.toml

```diff
@@ -40,6 +40,11 @@ walker = "^1.0.0"
 [target.x86_64-unknown-linux-gnu.dependencies.inotify]
 version = "^0.1"
 
+[target.x86_64-unknown-linux-gnu.dependencies.mio]
+git = "https://github.com/carllerche/mio.git"
```

Comment:
Thanks for looking that up/inquiring on IRC. Presumably 0.5.x will be branched off from master, so `EventedFd` should be included. I'll keep an eye on it either way.